lib/model/model.go

```diff
@@ -981,6 +981,20 @@ func (m *Model) GetIgnores(folder string) ([]string, []string, error) {
 	return lines, patterns, nil
 }
 
+func (m *Model) GetFolderFiles(folderID string) *db.FileSet {
```

Comment:
These look like dead code to me, but apparently not to the metalinter. Am I missing something?